frontend/src/pages/CourseSelector/main.less

```diff
@@ -63,4 +63,50 @@
   }
 }
 
+.collapsible-header {
```

Comment:
Could these css classes be included in a seperate less file? Maybe place these stylings into a file like `Collapsible.less` for now.